### PR TITLE
feat(useToast)!: rename useToast to useToastController

### PR DESCRIPTION
**Pull request by VividLemon**
# Describe the PR

A clear and concise description of what the pull request does.

## Small replication

A small replication or video walkthrough can help demonstrate the changes made. This is optional, but can help observe the intended changes. A mentioned issue that contains a replication also works.

## PR checklist

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix :bug: - `fix(...)`
- [ ] Feature - `feat(...)`
- [ ] ARIA accessibility - `fix(...)`
- [ ] Documentation update - `docs(...)`
- [ ] Other (please describe)

**The PR fulfills these requirements:**

- [ ] Pull request title and all commits follow the [**Conventional Commits**](https://www.conventionalcommits.org/) convention or has an [**override**](https://github.com/googleapis/release-please#how-can-i-fix-release-notes) in this pull request body **This is very important, as the `CHANGELOG` is generated from these messages, and determines the next version type. Pull requests that do not follow conventional commits or do not have an override will be denied**

### CODE REVIEW

File: apps/docs/src/components/TableOfContentsNav.vue [hunk 1]
```diff
@@ -147,7 +147,9 @@ const composablesList: {name: string}[] = [
   {name: 'useColorMode'},
   {name: 'useModal'},
   {name: 'useModalController'},
-  {name: 'useToast'},
+  {name: 'useToastController'},
+  {name: 'useTooltipController'},
+  {name: 'usePopoverController'},
```
Comment (VividLemon) [inline apps/docs/src/components/TableOfContentsNav.vue:152]: @xvaara Is this better off as a single composable which has `tooltip()` `popover()` methods?

File: packages/bootstrap-vue-next/src/components/BDropdown/BDropdown.vue [hunk 2]
```diff
@@ -83,22 +83,17 @@ import {BvTriggerableEvent} from '../../utils'
...
   boundary: 'clippingAncestors',
   boundaryPadding: undefined,
-  center: false,
   teleportTo: undefined,
   teleportDisabled: false,
   disabled: false,
-  dropend: false,
-  dropstart: false,
-  dropup: false,
-  end: false,
```
Comment (xvaara) [inline packages/bootstrap-vue-next/src/components/BDropdown/BDropdown.vue:101]: This is a breaking change from previous. And change from bsv2. But it's clearer to have them all the same.

File: packages/bootstrap-vue-next/src/types/ComponentOrchestratorTypes.ts [hunk 3]
```diff
@@ -36,38 +38,32 @@ export type ToastOrchestratorShowParam = {
   props?: MaybeRefOrGetter<PublicOrchestratedToast>
 }
 
-export type PrivateOrchestratedTooltip = Omit<BTooltipProps, 'modelValue' | 'reference'> & {
-  _reference: Exclude<BTooltipProps['reference'], undefined>
+export type PrivateOrchestratedTooltip = Omit<BTooltipProps, 'modelValue' | 'target'> & {
```
Comment (xvaara) [inline packages/bootstrap-vue-next/src/types/ComponentOrchestratorTypes.ts:41]: Tooltip/popover can have different target and reference (ie trigger tooltip from one element and show it related to another)

Comment (VividLemon) [reply packages/bootstrap-vue-next/src/types/ComponentOrchestratorTypes.ts:41]: Does one or both need to be set?

Which is which

File: packages/bootstrap-vue-next/src/components/BDropdown/BDropdown.vue (hunk 2 above)
Comment (VividLemon) [reply packages/bootstrap-vue-next/src/components/BDropdown/BDropdown.vue:101]: This is a fine breaking change imo that I added

File: apps/docs/src/docs/components/button-group.md [hunk 4]
```diff
@@ -138,13 +138,13 @@ that split dropdown menus are not supported when prop `vertical` is set.
 <HighlightCard>
   <BButtonGroup>
     <BButton>Button</BButton>
-    <BDropdown right text="Menu">
+    <BDropdown placement="right"text="Menu">
```
Comment (tbl0605) [inline apps/docs/src/docs/components/button-group.md:141]: There seems to be missing spaces between the 2 attributes ^^

File: packages/bootstrap-vue-next/src/types/ComponentOrchestratorTypes.ts (hunk 3 above)
Comment (xvaara) [reply packages/bootstrap-vue-next/src/types/ComponentOrchestratorTypes.ts:41]: if no reference is set, it's set to the same as target (the 99% use case). If it's changed, you need to rebind the popover.

Comment (VividLemon) [reply packages/bootstrap-vue-next/src/types/ComponentOrchestratorTypes.ts:41]: @xvaara Can you work on pushing to this branch for any changes you'd like to see for the composable & orchestrator? One issue that popped in my mind is how do we handle when a target component is unmounted but still in the composable stack... Not sure if the overhead of checking if the component is removed is worth it just lingering in the stack

Comment (VividLemon) [reply packages/bootstrap-vue-next/src/types/ComponentOrchestratorTypes.ts:41]: @xvaara On second thought, I'm just going to release 0.25 and not write any docs on it and keep it more or less "dark"

Comment (xvaara) [reply packages/bootstrap-vue-next/src/types/ComponentOrchestratorTypes.ts:41]: I'll take a look tonight. 

Can we add a onUnmount(() => remove(self)), to the popover/tooltip functions? I think that would cover most cases. Or are you meaning it's added to a another component, basically using a templateRef or dom query? Then we could add a mutation observer to the dom element, and remove it from the stack when it's removed from dom.

**Pull request merged**